index.html

```diff
@@ -1,12 +1,368 @@
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+                  </li>
+                  <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                        aria-label="Medium"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+                  </li>
+                </ul>
+              </section>
+            </section>
+          </section>
+    </div>
+
+    <!-- start works section -->
+
+    <section class="work_section">    
+
+        <!-- Start First card -->
+
+        <div class="middle_section">
+            <div class="pic">
+                <img src="Media/Snapshoot Portfolio.png" alt="this is a pic">
+            </div>
+            <div class="name">
+                <h1>
+                    Tonic
+                </h1>
+            </div>
+            <div class="descript flex">
+                <div class="camp">
+                    <h5>
+                        CANOPY
+                    </h5>
+                </div>
+                <div class="year">
+                    <ul class="list flex">
+                        <li>Back And Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="task">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="web flex">
+                <div class="right flex">
+                    <ul>
+                        <li>
+                            <a href="#">HTML</a>
+                        </li>
+                        <li>
+                            <a href="#">CSS</a>
+                        </li>
+                        <li>
+                            <a href="#">JavaScript</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="left"></div>
+            </div>
+            <div class="link flex">
+                <a href="#">
+                    See project
+                </a>
+            </div>
+        </div>
+
+        <!-- End of First card -->
+
+        <!-- Start Second card -->
+
+
+        <div class="middle_section">
+            <div class="pic">
+                <img src="Media/Snapshoot Portfolio1.png" alt="Snapshoot pic">
+            </div>
+            <div class="name">
+                <h1>
+                    Multi-Post Stories
+                </h1>
+            </div>
```

Comment:
- [ ] We only used one `h1` heading in the whole webpage in the hero section. the rest of the headings should be from `h2 to h6` following the best semantics. kindly consider fixing that in all the cards.
- [ ] Kindly consider applying `#172B4D` color to this heading in all the cards following figma guidelines.
- [OPTIONAL] First of all, I think you only need to take a div when there are multiple elements in that div. In your case there is only one `h` element so consider using it directly outside the div following best practices.